src/pages/steps/PlaceStep.tsx

```diff
@@ -157,13 +157,23 @@ const PlaceStep = ({
                           mainLanguage,
                         );
                         return (
-                          <Stack>
+                          <Stack
+                            css={`
+                              .address {
+                                color: ${({ theme }) => theme.colors.grey6};
```

Comment:
Is there a reason why you use an arrow function here?
You could simply write it like: 

```suggestion
                                color: ${theme.colors.grey6};
```

Reply:
Yes theme is not imported but injected via the arrow function (styled-components + own component library magic)